packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -1003,6 +1037,12 @@ export default class TabAdvanced extends Component {
                     <Toggle value={this._getIsStopBlocked()} onChange={this._onChangeBlockStop} />
                   </td>
                 </tr>
+                <tr>
+                  <th>{_('allowMigration')}</th>
```

Comment:
Revert the toggle to make it more in line with the others: _Block migration_

Reply:
Done